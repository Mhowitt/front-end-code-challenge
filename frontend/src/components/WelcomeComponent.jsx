import React from 'react';
import { Link } from 'react-router-dom';

const WelcomeComponent = props => {
  return (
    <div>
      <h1>Your Perfect Clothing Brand</h1>
      <button>
      <Link to="/profile">Let's find my perfect brand! </Link>
      </button>
    </div>
  )
}

export default WelcomeComponent;