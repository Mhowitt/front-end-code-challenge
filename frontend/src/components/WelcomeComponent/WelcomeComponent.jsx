import React from 'react';
import { Link } from 'react-router-dom';
import './WelcomeComponent.css';

const WelcomeComponent = props => {
  return (
    <div>
      <h2>Your Perfect Clothing Brand</h2>
      <div className="card-container">
        <p>
          {' '}
          Here at RightFit we believe that fashion should be tailored to you
          rather than the other way around
        </p>
        <div className="welcome-button-container">
          <p>Take our quiz to find which of our brand are designed for you</p>
          <Link to="/profile" className="welcome">
            <button className="start-quiz" onClick={props.handleClick}>
              Let's find my perfect brand!
            </button>
          </Link>
        </div>
      </div>
    </div>
  );
};

export default WelcomeComponent;
