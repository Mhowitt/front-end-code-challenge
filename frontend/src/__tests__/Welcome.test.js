import React from 'react'; // for JSX
import { shallow, mount } from 'enzyme'; // how to mount the component
import toJson from 'enzyme-to-json';
import WelcomeComponent from '../components/WelcomeComponent/Welcome.jsx';

describe('<WelcomeComponent />', () => {
  let wrapper;
  it('renders', () => {
    wrapper = shallow(<WelcomeComponent />);
  });
  it('matches snapshot', () => {
    const serialized = toJson(wrapper);
    expect(serialized).toMatchSnapshot();
  });
});
