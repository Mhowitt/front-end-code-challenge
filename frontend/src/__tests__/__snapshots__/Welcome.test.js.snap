// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WelcomeComponent /> matches snapshot 1`] = `
<div>
  <h2>
    Your Perfect Clothing Brand
  </h2>
  <div
    className="card-container"
  >
    <p>
       
      Here at RightFit we believe that fashion should be tailored to you rather than the other way around
    </p>
    <div
      className="welcome-button-container"
    >
      <p>
        Take our quiz to find which of our brand are designed for you
      </p>
      <Link
        className="welcome"
        replace={false}
        to="/profile"
      >
        <button
          className="start-quiz"
        >
          Let's find my perfect brand!
        </button>
      </Link>
    </div>
  </div>
</div>
`;
